src/templates/index.html

```diff
@@ -294,6 +294,20 @@ <h2 class="section__title">Meet some Members</h2>
             role="img" aria-label="Aimee Tacchi on LinkedIn" src="/img/linked_in.svg" class="social__item__img"></a>
         </li>
       </ul>
+    </div><div class="speaker">
+      <img src="<%= assetPath %>/img/chris-130.jpeg" class="speaker__img" alt="Photograph of Christopher Geary">
+      <h3>Christopher Geary</h3>
+      <p>Performance Evangelist</p>
+      <ul class="social">
+        <li class="social__item">
+          <a href="https://github.com/chrisgeary92" target="_blank"><img
```

Comment:
Can you add `rel="noopener"` to both of the `<a>` tags, if you take a look at the other cards you can see an example :)

`rel="noopener"` is essentially a security fix for links to pages you do not control (such as external sites).